src/covid19sim/base.py

```diff
@@ -482,7 +482,7 @@ def run(self, duration, outfile, start_time, all_possible_symptoms, port, n_jobs
                 self.tracker.track_risk_attributes(self.humans)
 
             # Let the hour pass
-            yield self.env.timeout(duration / TICK_MINUTE)
+            yield self.env.timeout(int(duration))
```

Comment:
why do we need it to be `int`?

Reply:
AFAIK, we don't, but since 1) no code in the simulation requires sub-second granularity 2) the current code assumed 2-minute/120-second time granules and 3) the timescales on which disease spreads are much longer than a second, I would like to enforce `int` timestamps.

Reply:
ok! Is this also done in `simulator.py` ? We are yielding there a lot as well.

Reply:
@pg2455 The only place in `simulator.py` with a non-`np.inf` timeout is exactly the one in `Human.at()` that is the subject of my report in Slack #p2p_simulator_repo.